docker/templates/Dockerfile.erb

```diff
@@ -155,18 +154,9 @@ ARG TARGETARCH
 COPY env2yaml/env2yaml-${TARGETARCH} /usr/local/bin/env2yaml
 # Place the startup wrapper script.
 COPY bin/docker-entrypoint /usr/local/bin/
-<% else -%>
-COPY scripts/config/pipelines.yml config/pipelines.yml
-COPY scripts/config/logstash.yml config/logstash.yml
-COPY scripts/config/log4j2.properties config/
-COPY scripts/config/log4j2.file.properties config/
-COPY scripts/pipeline/default.conf pipeline/logstash.conf
-RUN chown --recursive logstash:root config/ pipeline/
-# Place the startup wrapper script.
-COPY scripts/bin/docker-entrypoint /usr/local/bin/
-<% end -%>
 
 RUN chmod 0755 /usr/local/bin/docker-entrypoint
+<% end -%>
```

Comment:
Can we comment that this marks the end of the `else <% if image_flavor == 'ironbank' -%>` block?